perun-wui-profile/src/main/resources/cz/metacentrum/perun/wui/profile/client/resources/PerunProfileTranslation_cs.properties

```
@@ -27,6 +27,10 @@ addCert=Přidat nový certifikát
 
 loadingUserData=Načítám data o uživateli
 updateEmail=aktualizovat
+changePassword=změnit heslo
+resetPassword=resetovat heslo
```

Just about consistency - in the English version, you use the first upper case letter. The same thing at updateEmail.
```suggestion
changePassword=Změnit heslo
resetPassword=Resetovat heslo
```

---

Fixed also with translation for update email button.